encryption/README.md

```diff
@@ -1,22 +1,5 @@
 ## PREREQUISITE
 
-### GO
```

Comment:
kenapa yang Go dihapus Lif?

Reply:
itu gw ngapusin yg redundant, please check this out https://github.com/tokopedia/cryptography-example/tree/f_alif_example_python/encryption